staging/kos/pkg/apis/network/types.go

```diff
@@ -221,6 +221,11 @@ type NetworkAttachmentStatus struct {
 	// +optional
 	Errors NetworkAttachmentErrors
 
+	// SubnetCreationTime is the API server write time of the SubnetSectionSpec
+	// of the subnet identified by NetworkAttachmentSpec.Subnet.
+	// +optional
+	SubnetCreationTime metav1.MicroTime
```

Comment:
When we add metrics to track delays caused by controllers' downtime we'll want to add a `lastImplementationPrecursor` with the name of the last controller to start and the time of that start. For consistency (and better design) we'll want to get rid of this and have a more generic field like `lastClientWrite` that carries the type of the last client write (e.g. subnet creation, natt creation) and the time of that write. IMO those two fields should go in `ExtendedObjectMeta` rather than `Status`(although they are written by controllers rather than API servers). But for now this serves the intended purpose.

Reply:
Yes.